client/components/domains/registrant-extra-info/ca-form.jsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import {
+	camelCase,
+	difference,
+	isEmpty,
+	keys,
+	map,
+	pick,
+} from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getContactDetailsExtraCache } from 'state/selectors';
+import { updateContactDetailsCache } from 'state/domains/management/actions';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormCheckbox from 'components/forms/form-checkbox';
+
+const legalTypes = {
```

Comment:
So, I "lied", ended up doing the regular translation thing. Yeah, English and French are the obvious `ca` buyers but we're translating the strings elsewhere in this form using `translate` calls so we might as well do the selector values too IMO.